src/main/scala/com/raphtory/core/components/querytracker/QueryProgressTracker.scala

```diff
@@ -31,26 +32,16 @@ import scala.collection.mutable.ListBuffer
   *    : Returns the latest {s}`Perspective` processed by the query
   *
   *   {s}`getPerspectivesProcessed(): List[Perspective]`
-  *    : Returns list of perspectives processed for the query till the current timestamp
+  *    : Returns list of perspectives processed for the query so far
   *
   *   {s}`getPerspectiveDurations(): List[Long]`
-  *    : Returns the time duration taken to process each of perspectives till the current timestamp
+  *    : Returns the time duration taken to process each of perspectives
```

Comment:
`Returns the time taken to process each perspective in ...`
Can you add the unit?